### PR TITLE
add other kwarg to some loads

### LINKED ISSUE #1461: [inductor] AMP accuracy error - inception_v3
Repro
~~~

import torch
from torch import tensor, device
import torch.fx as fx
from torchdynamo.testing import rand_strided
from math import inf
from torch.fx.experimental.proxy_tensor import make_fx

# torch version: 1.13.0a0+git071f875
# torch cuda version: 11.6
# torch git version: 071f875046202b87213865dfc180abdf8368f116


# CUDA Info: 
# nvcc: NVIDIA (R) Cuda compiler driver 
# Copyright (c) 2005-2022 NVIDIA Corporation 
# Built on Thu_Feb_10_18:23:41_PST_2022 
# Cuda compilation tools, release 11.6, V11.6.112 
# Build cuda_11.6.r11.6/compiler.30978841_0 

# GPU Hardware Info: 
# NVIDIA A100-SXM4-40GB : 8 


from torch.nn import *
class Repro(torch.nn.Module):
    def __init__(self):
        super().__init__()

    
    
    def forward(self, cat_11, _to_copy_373):
        avg_pool2d_8 = torch.ops.aten.avg_pool2d.default(cat_11, [3, 3], [1, 1], [1, 1]);  cat_11 = None
        convolution_93 = torch.ops.aten.convolution.default(avg_pool2d_8, _to_copy_373, None, [1, 1], [0, 0], [1, 1], False, [0, 0], 1);  avg_pool2d_8 = _to_copy_373 = None
        return (convolution_93,)
        
args = [((2, 2048, 8, 8), (131072, 64, 8, 1), torch.float16, 'cuda'), ((192, 2048, 1, 1), (2048, 1, 1, 1), torch.float16, 'cuda')]
args = [rand_strided(sh, st, dt, dev) for (sh, st, dt, dev) in args]
mod = make_fx(Repro())(*args)

from torchinductor.compile_fx import compile_fx_inner
from torchdynamo.debug_utils import same_two_models

compiled = compile_fx_inner(mod, args)
assert same_two_models(mod, compiled, args, only_fwd=True), "Accuracy failed"
~~~

### PR DESCRIPTION
**Pull request by ngimel**
Fixes #1461. I tried to limit adding `other` kwarg, but it's still pretty broad, as it's unclear when this bug hits.

### CODE REVIEW

**Comment by ngimel** [comment]
Test errors are real, `other` branch doesn't like bool loads.

**Comment by jansel** [comment]
> Test errors are real, `other` branch doesn't like bool loads.

You should be able to use:
https://github.com/pytorch/torchdynamo/blob/2294e28537453bd4caa1a8e2572c3b04c6112ee9/torchinductor/graph.py#L100

**Review by jansel — APPROVED** [review]
